src/pages/index.js

```diff
@@ -143,9 +119,7 @@ function handleRemoveCard(data) {
         removeCardModal.close();
         data._handleDeleteButton();
```

Comment:
`_handleDeleteButton`

Please, delete the **low dash** from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.